app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/generator/GeneratorScreen.kt

```diff
@@ -636,7 +642,7 @@ private fun PasswordLengthSliderItem(
                 }
                 .testTag("PasswordLengthLabel")
                 .wrapContentWidth()
-                .width(labelTextWidth + 16.dp + 16.dp),
+                .width(labelTextWidth + 32.dp),
```

Comment:
Not sure if there is a specific reason to have it 16 + 16

Reply:
> // We want the width to be no wider than the label + 16dp on either side

Found this comment in history of the old repo. Not sure when it was lost, but I think adding it back would be helpful.